scripts/start-deployMagma

```diff
@@ -6,7 +6,7 @@ isDebugging && set -x
 
 resolveVersion
 
-if ! downloadUrl=$(get --json-path '$.link' "https://api.magmafoundation.org/api/v2/${VERSION}/latest"); then
+if ! downloadUrl=$(get --json-path '$.link' "https://api.magmafoundation.org/api/v2/${VERSION}/latest/${MAGMA_VERSION}"); then
```

Comment:
I know this script doesn't follow the pattern yet, but please do like most of the other scripts and pre-declare this variable and its default of being empty, such as

https://github.com/itzg/docker-minecraft-server/blob/a6c3943136f6c32e5a1e69488379917e330656c2/scripts/start-deployAutoCF#L7C6-L7C6